### ISSUE TITLE
Add icons for High-DPI monitors

### ISSUE THREAD

**Issue by boniatillo-com** [angelozerr/typescript.java#207]
To make a review of the icons and add the @2x version of them.

I am trying to setup the dev environment, if I have success I could work on this issue.

**Comment by boniatillo-com** [comment]
This is a screenshot with some good icons, provided by the Eclipse platform, and the too small icons, provided by TS IDE.

By the way, I do not have a 4k display but I can simulate the Eclipse hi-dpi support by adding this property to the eclipse.ini file:

```
-Dswt.autoScale=200
```

![image](https://user-images.githubusercontent.com/12187299/30395824-7c3271e6-9895-11e7-8ead-4badf0823b94.png)

**Comment by boniatillo-com** [comment]
Note that Eclipse scale the icons when are too small for the configured resolution, so the small icons gets pixelated.

**Comment by angelozerr** [comment]
@boniatillo-com any PR are welcome!

**Comment by boniatillo-com** [comment]
@angelozerr yes I am trying to setup de dev environment, tycho/maven is a pain for me!! but I have to face it soon or later :(
